src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
```

Comment:
```suggestion
import { MovieCard } from '../MovieCard;
```
там есть index через которые реэкспортируется компонент  и можно сократить запись